src/main/scala/magellan/Polygon.scala

```diff
@@ -209,6 +217,21 @@ class Polygon extends Shape {
 
   def getRing(index: Int): Int = indices(index)
 
+  def getVertexes():Array[Point]={
```

Comment:
can you make functions you need only within this class private? we can always widen the scope later if these functions are useful elsewhere